src/utils/train.py

```diff
@@ -64,7 +64,6 @@ def train(
         num_workers=config.num_workers,
         shuffle=True,
     )
-
```

Comment:
Try to minimize number of changes, this will help reviewers to get feedback faster. For example, avoid changes like this: there is no need to remove line here.